Examples/SlackClone/SlackChatMessageListViewController.swift

```diff
@@ -10,4 +10,106 @@ final class SlackChatMessageListViewController: ChatMessageListVC {
     override func cellContentClassForMessage(at indexPath: IndexPath) -> ChatMessageContentView.Type {
         SlackChatMessageContentView.self
     }
+
+    override func handleLongPress(_ gesture: UILongPressGestureRecognizer) {
+        let location = gesture.location(in: listView)
+        guard gesture.state == .began,
+              let indexPath = listView.indexPathForRow(at: location),
+              let cell = listView.cellForRow(at: indexPath) as? ChatMessageCell,
+              let messageContentView = cell.messageContentView as? SlackChatMessageContentView,
+              let message = messageContentView.content else {
+            return super.handleLongPress(gesture)
+        }
+
+        let reactionsLocation = gesture.location(in: messageContentView.slackReactionsView.collectionView)
+        guard let reactionsIndexPath = messageContentView.slackReactionsView.collectionView.indexPathForItem(at: reactionsLocation) else {
+            return super.handleLongPress(gesture)
+        }
+        
+        let reaction = messageContentView.slackReactionsView.reactions[reactionsIndexPath.row]
+        openReactionsSheet(for: message, with: reaction.type)
+    }
+
+    func openReactionsSheet(for message: ChatMessage, with reactionType: MessageReactionType) {
+        let messageController = client.messageController(cid: message.cid!, messageId: message.id)
+        let slackReactionListView = SlackReactionListViewController(
+            messageController: messageController,
+            reactionType: reactionType
+        )
+        present(slackReactionListView, animated: true)
+    }
+}
+
+/// Displays the reaction authors for a specific reaction type.
+class SlackReactionListViewController: UITableViewController, ChatMessageControllerDelegate {
+    let messageController: ChatMessageController
+    let reactionType: MessageReactionType
+
+    init(messageController: ChatMessageController, reactionType: MessageReactionType) {
```

Comment:
Feels weird needing to have a `ChatMessageController` per reaction type to only display reactions. IMO having a `ReactionListController` makes more sense.